OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -1670,3 +1677,20 @@ void testExternalForce()
             ASSERT_EQUAL(def, val, 10*accuracy);
     }
 }
+
+void testSerializeDeserialize() {
```

Comment:
Do you have any other model file in mind ?

The `ASSERT` is checking the property of the model read from the old file and new file.